serverManager/common/source/SessionServerApp.cpp

```diff
@@ -54,6 +53,25 @@ std::string getSessionServerPath()
     }
     return "/usr/bin/RialtoServer";
 }
+
+std::chrono::milliseconds getStartupTimeout()
+{
+    const char *customTimeout = getenv("RIALTO_SESSION_SERVER_TIMEOUT_MS");
+    std::chrono::milliseconds timeout{0};
```

Comment:
Do we want timeout to be disabled by default?
In my opinion, timeout should be disabled only for testing purposes, so we will always have to set it in production code.